test/ffmpeg-qsv/encode/hevc.py

```diff
@@ -282,7 +283,7 @@ def test(self, case, bitrate, maxrate, fps, maxframesize, profile):
     self.encode()
 
 class pict(HEVC8EncoderTest):
-  def init(self, tspec, case, gop, bframes, bitrate, qp, maxrate, profile, rcmode):
```

Comment:
I didn't mean to imply we should change the `pict` test class.  Rather, we should copy it and name it for a new `seek` test class.